### ISSUE TITLE
Images in documentation are broken

### ISSUE THREAD

**Issue by jabbett** [radiant/radiant#394]
Getting Started, Pages, Layouts, Snippets... all have invalid image URLs.

![image](https://cloud.githubusercontent.com/assets/923971/9935751/68c9b4fe-5d26-11e5-8569-eab8480c5894.png)

**Comment by Paladin** [comment]
Yep. Upon inspection they (or at least the one from above, I didn't do a systematic search, yet) appear not to be github-hosted images, but rather image links into Skitch that no longer work. Anyone have the originals handy (or access into the skitch account where they might be located)?
